YDotNet/Native/Types/Branches/BranchKind.cs

```diff
@@ -1,11 +1,14 @@
+using YDotNet.Document.Cells;
+
 namespace YDotNet.Native.Types.Branches;
 
 internal enum BranchKind
 {
-    Null = 0,
-    Array = 1,
-    Map = 2,
-    Text = 3,
-    XmlElement = 4,
-    XmlText = 5,
+    Null = OutputTag.Null,
```

Comment:
I don't like this, because it creates a circular dependency between Native namespace and the higher level Document  namespace. 

We defined, that Document depends on Native, not the other direction.

Reply:
That makes sense, I actually missed this detail when I was applying this change. My goal here was to reduce the duplication of these constants in the project.

But, after all, I guess we'll have to keep them. I just pushed [this commit](https://github.com/y-crdt/ydotnet/commit/94dc35c91635d3b8f0415a1e6dc4cdb051829064) to revert this change.

Reply:
@SebastianStehle I'm resolving this because this problem is now gone.